msal/src/main/java/com/microsoft/identity/client/SingleAccountPublicClientApplication.java

```diff
@@ -93,39 +224,168 @@ public void onTaskCompleted(List<ICacheRecord> cacheRecords) {
                     @Override
                     public void onError(Exception exception) {
                         callback.onError(exception);
+
                     }
                 });
     }
 
+    private void checkCurrentAccountNotifyCallback(final CurrentAccountCallback callback, List<ICacheRecord> newAccountRecords){
+        IAccount localAccount = getPersistedCurrentAccount();
+        IAccount newAccount = newAccountRecords == null ? null : getAccountFromICacheRecordList(newAccountRecords);
+
+        if (localAccount == null) {
```

Comment:
put this in didExistingCurrentAccountChange()

Reply:
I didn't follow this comment.  Let's discuss in person.

Reply:
Oh. It's just that they are doing the same operation.